app/src/main/java/com/bucketlist/dao/ILocalBucketListItemDAO.kt

```diff
@@ -10,11 +10,11 @@ interface ILocalBucketListItemDAO {
     fun getAll()  : LiveData<List<BucketListItem>>
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
-    fun insertAll(bucketlistitems: BucketListItem)
+    fun insertAll(bucketListItems: BucketListItem)
```

Comment:
changing naming convention to use camel case as recommended here: https://kotlinlang.org/docs/coding-conventions.html#function-names

Reply:
Good catch.